### ISSUE TITLE
Erata in pybuda First 5 steps install tt_buda setup. 

### ISSUE THREAD

**Issue by danindiana** [tenstorrent/tt-buda-demos#127]
![Selection_255](https://github.com/user-attachments/assets/3e2a3edd-f8a8-4483-94bb-434e050b912a)

For: https://github.com/tenstorrent/tt-buda-demos/blob/main/first_5_steps/1_install_tt_buda.md

Issues when attempting to pip install pybuda first. Should attempt to install torchvision, then tvm, and finally pybuda.

Instructions show:
![image](https://github.com/user-attachments/assets/da2c6afc-079c-4959-bd24-def49caa842c)

Step 5. Pip install PyBuda, TVM and Torchvision whl files
pip install --upgrade pip==24.0
pip install pybuda-<version>.whl tvm-<version>.whl torchvision-<version>.whl

Should proceed like the following:
$pip install --upgrade pip
$unzip pybuda-gs-v0.19.3-ubuntu-22-04-amd64-python3.10.zip
$:pip install tvm-0.14.0+dev.tt.0840d6bef-cp310-cp310-linux_x86_64.whl 
$:pip install torchvision-0.16.0+fbb4cc5-cp310-cp310-linux_x86_64.whl
$:pip install pybuda-0.1.240917+dev.gs.12bb84b-cp310-cp310-linux_x86_64.whl

**System (please complete the following information):**
![baruch@spinoza: ~-programs-tens_001](https://github.com/user-attachments/assets/e8a78685-03b7-4281-88d7-f29315e8ca3a)

**Additional context**

python --version
Python 3.10.12

OS: Ubuntu 22.04 jammy
 Kernel: x86_64 Linux 6.8.0-47-generic
 Uptime: 19m
 Packages: 2033
 Shell: bash 5.1.16
 Disk: 27G / 1.8T (2%)
 CPU: AMD Ryzen 7 7700 8-Core @ 16x 5.389GHz
 GPU: AMD/ATI
 RAM: 3320MiB / 63437MiB

**Comment by danindiana** [comment]
Smoke test. Looks good. 
![programs-tens-pybudvenv_001](https://github.com/user-attachments/assets/7f76b481-d223-4e2e-bc28-b0c818b852ed)


![programs-tens-pybudvenv_002](https://github.com/user-attachments/assets/d2a96fd9-18fc-4f0d-a803-d0912df52b43)

**Comment by milank94** [comment]
@danindiana to clarify, are you seeing that issue show up when you do the single line pip instal of the dependencies? i.e.

```bash
pip install --upgrade pip
pip install pybuda-0.1.240917+dev.gs.12bb84b-cp310-cp310-linux_x86_64.whl pip install torchvision-0.16.0+fbb4cc5-cp310-cp310-linux_x86_64.whl pip install tvm-0.14.0+dev.tt.0840d6bef-cp310-cp310-linux_x86_64.whl
```

**Comment by cekim7** [comment]
Hi, this is the issue I addressed in my pull request #129
